model-engine/README.md

```diff
@@ -41,3 +41,9 @@ Run `mypy . --install-types` to set up mypy.
 
 Most of the business logic in Model Engine should contain unit tests, located in
 [`tests/unit`](./tests/unit). To run the tests, run `pytest`.
+
+Make sure to set a WORKSPACE env var that points to the root directory of this repo
```

Comment:
do we actually need this?

Reply:
the code requires a WORKSPACE env to be set for things to work.

Reply:
nevermind, i decided to just add a mock for the unit test case

Reply:
Nevermind, decided to add mocks to handle it in the unittest directly